.circleci/config.yml

```diff
@@ -201,9 +201,21 @@ workflows:
       # https://github.com/spotbugs/spotbugs-maven-plugin/issues/806 explains
       # why we need execution of spotbugs without any other maven plugins which can change binaries
       - validate-with-maven-script:
-          name: "spotbugs-and-pmd"
+          name: "spotbugs"
           image-name: *cs_img
-          command: "./.ci/validation.sh spotbugs-and-pmd"
```

Comment:
assuming compiling takes about 30 sec. its worth to apply SOC and give more focus. Build takes anyways more then 30 mins so it might be possible to give a try.